src/taskFunctions.js

```diff
@@ -0,0 +1,29 @@
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const addTask = (tasks, description) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  tasks.push(task);
+  saveTasks(tasks);
+  return tasks;
+};
+
+export const deleteTask = (tasks, index) => {
+  tasks.splice(index, 1);
+  for (let i = index; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  saveTasks(tasks);
+  return tasks;
+};
```

Comment:
- [ ] Just as highlighted by the previous reviewer, you're required to update the tasks' indexes to be in order from 1...4 after any task is deleted. Currently, the index of the tasks is not being updated. Kindly work on that 👍🏽

<img width="543" alt="Screenshot 2023-08-15 at 9 16 16 PM" src="https://github.com/gdvl777/to_do_list_ok/assets/50941074/ea2bb7ad-ebb7-4a53-bdd8-b76c23784a54">

Reply:
[https://www.loom.com/share/b538242b76394d0b830aaa40264d449b](url)
Please watch the video, I show that the index of the tasks IS BEING UPDATED

Reply:
> * [ ]  Just as highlighted by the previous reviewer, you're required to update the tasks' indexes to be in order from 1...4 after any task is deleted. Currently, the index of the tasks is not being updated. Kindly work on that 👍🏽
> 
> <img alt="Screenshot 2023-08-15 at 9 16 16 PM" width="543" src="https://user-images.githubusercontent.com/50941074/260845992-ea2bb7ad-ebb7-4a53-bdd8-b76c23784a54.png">
[https://www.loom.com/share/b538242b76394d0b830aaa40264d449b](url) 
Please watch the video, I show that the index of the tasks IS BEING UPDATED